content/geometry/ConvexHull.h

```diff
@@ -1,45 +1,36 @@
 /**
- * Author: Johan Sannemo
- * Date: 2017-04-16
- * License: CC0
- * Source: Basic algorithm knowledge
+ * Author: Stjepan Glavina, chilli
+ * Date: 2019-05-05
+ * License: Unlicense
+ * Source: https://github.com/stjepang/snippets/blob/master/convex_hull.cpp
  * Description:
 \\\begin{minipage}{75mm}
 Returns a vector of indices of the convex hull in counter-clockwise order.
 Points on the edge of the hull between two other points are not considered part of the hull.
+Works with integers.
```

Comment:
This is pretty obvious, the question is whether it works with doubles. I would imagine it does?

Reply:
Shouldn't geometry algorithms that work on integers be a superset of those that work on doubles? It's obvious if you understand what the algorithm is doing/what operations it's using, but I think it's good to make clear.

Reply:
The only possible source of numerical error is the operation `h[t-2].cross(h[t-1], p) <= 0`. And that's as good as it gets.

Reply:
Also, I removed that from the description.